### ISSUE TITLE
Code Table Request - New part name midbody

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#3145]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

Goal
Help a new collection get into Arctos

Context
New terms/combinations of terms.

> Aplacophoran molluscs are honorary worms whose head and tail ends are taxonomically informative whereas the middle region of the body is uniform and boring. We often cut them in thirds and put the head and tail in the same container of formalin / glutaraldehyde and the “midbody” region would just be the middle third or less that would potentially be further divvied out into ethanol, RNAlater, and/or cryo. If “head, tail” is ok can I please have “midbody” too?

Table
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name

Value
midbody
midbody (frozen) 
midbody (RNAlater)

Definition
Middle third of the body. Preservation unknown

Collection type
Inv

Attribute data type
N/A

Attribute value
N/A

Attribute units
N/A

Part tissue flag
yes

Other ID BaseURL
N/A

Priority
Please assign a priority-label.

**Comment by ebraker** [comment]
would "preservation unknown" be part of this definition (and others) you've recently requested? Seems unnecessary to include it in the part definition.

**Comment by dustymc** [comment]
I like being explicit, mostly for the data entry folks.

**Comment by Jegelewicz** [comment]
> would "preservation unknown" be part of this definition (and others) you've recently requested? Seems unnecessary to include it in the part definition.

Agree with Dusty. It does seem unnecessary, but also it adds clarity for those students entering data. Of course we aren't being consistent at all and SOOOOOOOOO many parts have no definition, so there's that.

**Comment by Jegelewicz** [comment]
Add midbody but use part preservation.

**Comment by Jegelewicz** [comment]
midbody added.

**Comment by Jegelewicz** [comment]
I supposedly added this part name, but I don't see it in the code table, neither do I find it in the recent part/preservation separation scheme. (see https://docs.google.com/spreadsheets/d/1AY1EXQzKUSAg9EvZFjayTXcCBciXd_syCxt0GSX2tww/edit?usp=sharing)

@dustymc did I ever add it? If so, what the heck did we decide to do here?

**Comment by dustymc** [comment]
I deleted all unused parts somewhere during that migration, probably got it there.

FWIW adding not-parts to our parts list sounds like an even worse idea now that we've made significant progress in the opposite direction. This is a condition, __maybe__ an "anatomical direction," I can't understand how it's a part.

**Comment by Jegelewicz** [comment]
Yeah - ALMNH:Inv will now have things identified as "midbody" so we need to figure out the best path forward for that stuff.

Here is what I suggest:

part name = tissue
part remark = midbody
part attribute type = "anatomical direction reference"
part attribute value = mid

There are other paths that could work - @kmkocot how would this work best for you?
